snakeAndLadderApp/src/com/workAtTech/snakeAndLadderApp/SnakeAndLadderAppLogic.java

```diff
@@ -0,0 +1,104 @@
+package com.workAtTech.snakeAndLadderApp;
+
+import com.workAtTech.snakeAndLadderApp.model.Board;
+import com.workAtTech.snakeAndLadderApp.model.Ladder;
+import com.workAtTech.snakeAndLadderApp.model.Player;
+import com.workAtTech.snakeAndLadderApp.model.Snake;
+
+import java.util.*;
+
+public class SnakeAndLadderAppLogic {
```

Comment:
Use a better class name

Reply:
Thank You @gcnit. Changed SnakeAndLadderAppLogic class name to SnakeAndLadderAppService.